Status: distro-specific

# In a distro, we can't rely on compile time detection of these.
# If we have to ship a config, it would be with these disabled anyway.

--- a/lib/direct/ppcasm_memcpy_cachable.S
+++ b/lib/direct/ppcasm_memcpy_cachable.S
@@ -34,8 +34,6 @@
 
 #define __ASSEMBLY__
 
-#include <linux/config.h>
-
 #if defined(CONFIG_8xx) || defined(CONFIG_403GCX)
 #define L1_CACHE_LINE_SIZE       16
 #define LG_L1_CACHE_LINE_SIZE     4 
